easybuild/easyconfigs/g/GADMA/GADMA-2.0.0rc12-foss-2020b.eb

```diff
@@ -0,0 +1,65 @@
+# Author: Jasper Grimm
+
+easyblock = 'PythonBundle'
+
+name = 'GADMA'
+version = '2.0.0rc12'
+
+homepage = 'https://gadma.readthedocs.io/en/latest/index.html'
+description = """Genetic Algorithm for Demographic Model Analysis"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+builddependencies = [('CMake', '3.18.4')]
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('matplotlib', '3.3.3'),
+    ('Pillow', '8.0.1'),
+    ('IPython', '7.18.1'),
+    ('h5py', '3.1.0'),
+    ('scikit-allel', '1.3.2'),
+    ('networkx', '2.5'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('attrs', '21.2.0', {   # demes needs attrs >= 20.3.0
+        'modulename': 'attr',
+        'checksums': ['ef6aaac3ca6cd92904cdd0d83f629a15f18053ec84e6432106f7a4d04ae4f5fb'],
+    }),
+    ('ruamel.yaml.clib', '0.2.2', {
+        'modulename': False,
+        'checksums': ['2d24bd98af676f4990c4d715bcdc2a60b19c56a3fb3a763164d2d8ca0e806ba7'],
+    }),
+    ('ruamel.yaml', '0.17.4', {
+        'checksums': ['44bc6b54fddd45e4bc0619059196679f9e8b79c027f4131bb072e6a22f4d5e28'],
+    }),
+    ('nlopt', '2.6.2', {
+        'checksums': ['21f10b0bbcf32b4d370c371e967c01df50ce96c258f483a32b9140b57e9e410d'],
+    }),
```

Comment:
The failure that I am seeing is during the building of `nlopt`: https://gist.github.com/branfosj/a59ab25d0813dac9fa6455eb9574bcba

The error is unclear, but I think there is a missing dependency.

Reply:
Can you try building the NLopt-python easyconfig and see if that works?